tools/dockerfile/grpc_clang_format/clang_format_all_the_things.sh

```diff
@@ -16,7 +16,7 @@
 set -e
 
 # directories to run against
-DIRS="src/core/lib src/core/tsi src/core/ext src/cpp test/core test/cpp include src/compiler src/csharp src/ruby third_party/address_sorting src/objective-c"
+DIRS="src/core/lib src/core/tsi src/core/ext/filters src/core/ext/transport src/cpp test/core test/cpp include src/compiler src/csharp src/ruby third_party/address_sorting src/objective-c"
```

Comment:
Instead of restricting the set of directories under ext that get checked, let's change the `find` command below to exclude files ending in `.upb.h` or `.upb.c`.  That way, we won't be accidentally skipping this check if we add a new directory under ext in the future.